src/index.html

```diff
@@ -5,8 +5,363 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo Moyo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">SMARTPHONES</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+            data-qa="nav-hover">LAPTOPS & COMPUTERS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">GADGETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">TABLETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">PHOTO</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div data-qa="card" class="card">
+        <img
+          class="card__img" src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
```

Comment:
```suggestion
        <img
          class="card__img" 
          src="../src/images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
        >
```
Pay attention to the formatting